docs/UserGuide.md

```diff
@@ -756,19 +799,22 @@ $$
 
 ##### Examples:
 
-* (assuming a list of persons is currently shown) `owe 1 m/10.00`
+* `owe 1 m/10.00`
 
 ##### Outcome:
 
 For `owe 1 m/10.00`:
 <center>
-<img src="images/UG-Basics/owe-output-1.png" width=700>
+<img src="images/UG-Basics/owe-output-1.png" >
 </center>
 
 ##### Possible Failures:
 
-If the entry does not exist, the command will fail. The command will also reject
-non-numeric amounts of money, or money that exceeds 2 decimal places.
+
+- The [_Index of person_](#glossary) must be within range. I.e., it must be greater than 0, and smaller than the length of the currently displayed list of people. This is because the [_Index of person_](#glossary) will be used to indicate which person to operate the command on.
++ The amount of money provided `m/` must also be numeric and in dollars, and cannot exceed 2 decimal places. I.e., the following are invalid for the amount:
```

Comment:
This is too long and repetitive/droning and reads like an informal specification instead of a UG
```suggestion
- The [_Index of person_](#glossary) must be within range.
+ The amount of money provided `m/` must also be numeric and in dollars, and cannot exceed 2 decimal places. For example, the following are invalid for the amount:
```